Travis build and local build report differ with version 0.1.0

>> [ERROR] ERROR found by checkstyle: .binding.avmfritz.test/META-INF/MANIFEST.MF:25 The MANIFEST.MF file must not contain any Require-Bundle entries. Instead, Import-Package must be used.
> 
> The "Require-Bundle" entries in the test fragment (for org.junit, org.mockito, org.hamcrest) are the only exception from this rule. We have to adopt this in the checkstyle configuration.

See openhab/openhab2-addons#2516 .

--- a/src/main/resources/rulesets/checkstyle/suppressions.xml
+++ b/src/main/resources/rulesets/checkstyle/suppressions.xml
@@ -5,7 +5,7 @@
 
 <suppressions>
     <!-- These suppressions define which files to be suppressed for which checks. -->
-    <suppress files=".+\\internal\\.+\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
+    <suppress files=".+[\\/]internal[\\/].+\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
     <suppress files=".+DTO\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck" />
     <suppress files=".+Impl\.java" checks="JavadocType|JavadocVariable|JavadocMethod|JavadocFilterCheck"/>
     <!-- The WiFi led binding contains test packages that shouldn't be included in the binary build -->
@@ -15,7 +15,7 @@
     <!-- IO and Voice bundles have specific use cases - they use only the config .xml files and will log false positives as well -->
     <suppress files=".+org.openhab.binding.homematic.+|.+org.openhab.binding.tellstick.+|.+org.openhab.io.+|.+org.openhab.voice.+" checks="EshInfXmlCheck"/>
     <!-- All generated files will skip the author tag check -->
-    <suppress files=".+\\gen\\.+\.java" checks="AuthorTagCheck"/>
+    <suppress files=".+[\\/]gen[\\/].+\.java" checks="AuthorTagCheck"/>
     <!-- Some checks will be supressed for test bundles -->
-    <suppress files=".+.test\\.+" checks="RequireBundleCheck"/>
+    <suppress files=".+.test[\\/].+" checks="RequireBundleCheck"/>
 </suppressions>

